types/validator_set.go

```diff
@@ -94,6 +97,20 @@ func (vals *ValidatorSet) IncrementProposerPriority(times int) {
 	vals.Proposer = proposer
 }
 
+/*
+
+func (vals *ValidatorSet) RescalePriorities() {
+	tvp := vals.TotalVotingPower()
+	diff := computeMaxMinPriorityDiff(vals)
+	if diff > K * tvp {
+		ratio := float64(K * tvp) / float64(diff)
```

Comment:
There were some concerns around on using floats (probably on slack, I'll see if I can dig those up).

Reply:
Ok, please let me know. This will most likely be considered under https://github.com/tendermint/tendermint/pull/3182/
I just did some experimentation here. The main objective was to remove the diff from the parameters and contain the priority window size  here. Not sure the caller should decide on this.

Reply:
removed